code/game/objects/structures/stool_bed_chair_nest/bed.dm

```diff
@@ -173,7 +173,7 @@
 	..(newloc,"wood","leather")
 
 /obj/structure/bed/padded/New(var/newloc)
-	..(newloc,"plastic","cotton")
+	..(newloc,DEFAULT_WALL_MATERIAL,"cotton")

```

Comment:
Why change it from plastic to steel?

Reply:
Makes them in line with the rest of the furniture.